suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -117,6 +117,10 @@ export const prepareAccountsReducer = createReducerWithExtraDeps(
 
 export const selectAccounts = (state: AccountsRootState) => state.wallet.accounts;
 
+export const selectUserHasAccounts = memoize((state: AccountsRootState): boolean =>
```

Comment:
no need to memoize if return value is boolean are there will be only few accounts